scripts/integration/mqtt/test.yaml

```diff
@@ -0,0 +1,8 @@
+args:
```

Comment:
Thanks for using the new integration test setup, by the way.

Reply:
That's a pretty nice setup to use, honestly. Without prior knowledge, I successfully launched the test locally on the 2nd try. I think that's a high reward :)